cassandra_backup_service.py

```diff
@@ -246,6 +246,9 @@ def append_cqlsh_args(cmd, args):
     if args.cqlsh_user:
         cmd.append('-p')
         cmd.append(args.cqlsh_pass)
+    if args.endpoint_url:
+        cmd.append('--endpoint-url')
+        cmd.append(args.endpoint_url)
```

Comment:
This is not cql argument, please remove it

Reply:
Removed: https://github.com/onzra/cassandra_backup_service/pull/53/commits/cd0ffd6a4b62050a23a48c13247fc3d57be5ff56

Reply:
Thanks @bogdan3d ! Could you please merge and tag the release

Reply:
@vikmalik done!